sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala

```diff
@@ -351,6 +353,62 @@ class TPCDSModifiedPlanStabilityWithStatsSuite extends PlanStabilitySuite with T
   }
 }
 
+trait V2TPCDSBase extends TPCDSBase {
+  override protected def sparkConf: SparkConf =
+    super.sparkConf.set("spark.sql.catalog.testcat", classOf[InMemoryCatalog].getName)
```

Comment:
seems this is testing in memory v2 table, but not parquet v2?

Reply:
You are right. Admittedly, I'm a bit lost when it comes to DSv2...
I've just added `ParquetV2TableCatalog` for this test in: https://github.com/apache/spark/pull/38640/commits/fc9d086c10499446ec5f095368a1c06e803e9455